modules/oe_webtools_media/tests/src/FunctionalJavascript/MediaSourceWebtoolsTest.php

```diff
@@ -287,4 +345,23 @@ protected function getMediaByName(string $name): MediaInterface {
     return reset($entities);
   }
 
+  /**
+   * Mocks the http-client.
+   *
+   * @param \GuzzleHttp\Psr7\Response ...$responses
+   *   Several number of responses.
+   */
+  protected function mockHttpClient(Response ...$responses): void {
```

Comment:
I don't think this is needed, since it's not used in the test above.

Reply:
Indeed, leftovers of previous tries... :(